recurly/client.py

```diff
@@ -820,6 +828,10 @@ def remove_shipping_address(self, account_id, shipping_address_id):
             Shipping Address ID.
 
 
+        Returns
+        -------
+        Empty
```

Comment:
These previously void methods on the client now return an `Empty` resource. This allows you to get the response metadata even though there was no body:

```python
response = client.remove_line_item("1234").get_response()
```